app/app/settings.py

```diff
@@ -104,6 +111,22 @@
     }
 }
 
+# toolbar settings
+if DEBUG:
+
+    def show_toolbar(request):
+        return DEBUG
+
+    DEBUG_TOOLBAR_CONFIG = {
+        "IS_RUNNING_TESTS": False,
+        "SHOW_TOOLBAR_CALLBACK": "app.settings.show_toolbar",
```

Comment:
This disables the protection of the default `debug_toolbar.middleware.show_toolbar` that only responds to IP addresses in `INTERNAL_IPS`. Is it required for Docker, or why do we need to remove that?